app/src/main/java/org/simple/clinic/drugs/search/DrugsSearchScreen.kt

```diff
@@ -40,6 +43,9 @@ class DrugsSearchScreen : BaseScreen<
     DrugSearchEffect,
     Unit>(), DrugSearchUi, UiActions {
 
+  @Inject
+  lateinit var activity: AppCompatActivity
```

Comment:
Why are we inject activity here?

Reply:
I had initially implemented the screen as an activity. #Mistake 😅 Altho, it led to me understanding the navigation framework more clearly. Which was great. :P